src/main/java/edu/cornell/kfs/module/purap/jaggaer/contract/xml/Header.java

```diff
@@ -0,0 +1,56 @@
+package edu.cornell.kfs.module.purap.jaggaer.contract.xml;
+
+import java.util.Date;
+
+import edu.cornell.kfs.sys.xmladapters.ZonedStringToJavaDateXmlAdapter;
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlRootElement;
+import jakarta.xml.bind.annotation.XmlType;
+import jakarta.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = {
+    "messageId",
+    "timestamp",
+    "authentication"
+})
+@XmlRootElement(name = "Header")
+public class Header {
```

Comment:
Supplier has a Header object as well, it contains all the fields here, plus relatedMessageId.  Can supplier and contract share?

Reply:
I think there are too many differences between the two Header DTOs at this time. We may or may not be fully consolidating the Authentication sub-DTO (see the other comment), and my version of the DTO converts the timestamp string into a Java Date.

We could possibly look into consolidating the two DTOs at a future date.